### PR TITLE
Booking optimization - rendering date range picker and enabled user to select date range

### PR DESCRIPTION
**Pull request by insivika**

### CODE REVIEW

File: client/src/components/Booker.js [hunk 1]
```diff
@@ -60,36 +61,82 @@ class Booker extends React.Component {
...
-        onChange={this.handleChange}
-        excludeDates={this.state.bookedDates}
+          selected={this.state.startDate}
+          selectsStart
+          excludeDates={this.state.bookedDates}
+          minDate={new Date()}
+          maxDate={this.addMonths(new Date(), 5)}
+          startDate={this.state.startDate}
+          endDate={this.state.endDate}
+          onChange={this.handleChangeStart}
```
Comment (allenhsieh) [inline client/src/components/Booker.js]: is this working? do you have to bind this somewhere?

File: client/src/components/Booker.js [hunk 2]
```diff
@@ -60,36 +61,82 @@ class Booker extends React.Component {
...
-      <p>{this.state.message}</p>
+      <DatePicker
+          selected={this.state.endDate}
+          selectsEnd
+          excludeDates={this.state.bookedDates}
+          minDate={new Date()}
+          maxDate={this.addMonths(new Date(), 5)}
+          startDate={this.state.startDate}
+          endDate={this.state.endDate}
+          onChange={this.handleChangeEnd}
```
Comment (allenhsieh) [inline client/src/components/Booker.js]: same question as before, i'm used to binding functions

File: client/src/components/Booker.js (hunk 1 above)
Comment (insivika) [reply client/src/components/Booker.js]: It's bound via an arrow function

File: client/src/components/Booker.js (hunk 2 above)
Comment (insivika) [reply client/src/components/Booker.js]: I like to bind them via arrow functions

**Pull request merged**